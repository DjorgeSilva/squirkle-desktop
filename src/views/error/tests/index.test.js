// @flow

import React from 'react'
import { shallow } from 'enzyme'
import Error from '../'

describe('Error', (): void => {
  describe('When mounted', (): void => {
    let wrapper

    beforeAll((): void => {
      wrapper = shallow(<Error />)
    })

    it('renders correctly', (): void => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
