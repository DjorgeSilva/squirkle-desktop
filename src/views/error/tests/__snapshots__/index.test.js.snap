// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error When mounted renders correctly 1`] = `
<div
  className="errorViewContainer"
>
  <h2>
    Error
  </h2>
</div>
`;
